applications/FluidDynamicsApplication/test_examples/cyl_bench/run_test.py

```diff
@@ -178,9 +178,6 @@ def PrintResults(model_part):
         velocity_linear_solver = SkylineLUFactorizationSolver()
     elif ProjectParameters.Velocity_Linear_Solver == "Super LU":
         velocity_linear_solver = SuperLUSolver()
-    elif ProjectParameters.Velocity_Linear_Solver == "Parallel MKL Pardiso":
```

Comment:
my suggestion would be to remove completely this test. It is old and does not match the modern testing interface. I completely agree that this should be done as a part of a different PR